README.rst

```diff
@@ -1,6 +1,22 @@
 ftw.referencewidget
-==============
+===================
 
+This Widget provides a Referencebrowser and a searchfield, which allows users to select references.
+
+The basequery is all types which are not in the types_not_searched property all modification to the allowed types are relative to this query.
```

Comment:
I disagree with this behavior: referencing things has nothing to do with searching things. I might have a situation where I explicitly want to reference a thing which I explicitly do not want to be found via the search.

I would make this more general and allow to traverse / "open" all folderish types (including content pages) and make all item selectable. Everything else must be widget configuration and I see no "saner" defaults than this.

The behavior of the "normal" navigation and of the search has nothing to do with backend / editing work where the editor must be allowed to see more things and must have better insight in the internal structure.